client/components/search-form.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+const apiId = '0e3da9a8';
+const apiKey = '931c76f72bf3477f440a0f37e67e6ddc';
```

Comment:
You should not commit your API keys to your code. Once you have fixed and committed the other couple notes, please lemme know